MetricsUploader/MetricsUploaderWindowsTest.cpp

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include "MetricsUploader/MetricsUploader.h"
+
+using orbit_metrics_uploader::MetricsUploader;
+
+TEST(MetricsUploader, CreateMetricsUploaderFromEmptyClient) {
+  auto metrics_uploader = MetricsUploader::CreateMetricsUploader("MetricsUploaderEmptyClient");
+  EXPECT_EQ(metrics_uploader.has_value(), false);
+}
+
+TEST(MetricsUploader, CreateMetricsUploaderFromStartClient) {
+  auto metrics_uploader = MetricsUploader::CreateMetricsUploader("MetricsUploaderStartClient");
+  EXPECT_EQ(metrics_uploader.has_value(), false);
+}
+
+TEST(MetricsUploader, CreateMetricsUploaderFromSendEventClient) {
+  auto metrics_uploader = MetricsUploader::CreateMetricsUploader("MetricsUploaderSendEventClient");
+  EXPECT_EQ(metrics_uploader.has_value(), false);
+}
+
+TEST(MetricsUploader, StartMetricsUploaderWithError) {
+  auto metrics_uploader =
+      MetricsUploader::CreateMetricsUploader("MetricsUploaderStartWithErrorClient");
+  EXPECT_EQ(metrics_uploader.has_value(), false);
+}
+
+TEST(MetricsUploader, SendLogEvent) {
+  auto metrics_uploader =
+      MetricsUploader::CreateMetricsUploader("MetricsUploaderStartAndSendEventClient");
+  EXPECT_EQ(metrics_uploader.has_value(), true);
+  bool result = metrics_uploader.value()->SendLogEvent(
+      orbit_metrics_uploader::OrbitLogEvent_LogEventType_UNKNOWN_EVENT_TYPE);
+  EXPECT_EQ(result, false);
+  result = metrics_uploader.value()->SendLogEvent(
+      orbit_metrics_uploader::OrbitLogEvent_LogEventType_ORBIT_INITIALIZED);
+  EXPECT_EQ(result, true);
+  result = metrics_uploader.value()->SendLogEvent(
+      orbit_metrics_uploader::OrbitLogEvent_LogEventType_ORBIT_CAPTURE_DURATION, 100);
+  EXPECT_EQ(result, true);
+}
+
+TEST(MetricsUploader, SecondMetricsUploader) {
```

Comment:
What do you mean by `SecondMetricsUploader`?

Reply:
In this test I want to check that it's not possible to create MetricsUploader for the second time. Renamed this test